src/main/drivers/sdcard_spi.c

```diff
@@ -70,18 +67,18 @@ static bool sdcardSpi_isFunctional(void)
 
 static void sdcard_select(void)
 {
-    SET_CS_LOW;
+    IOLo(sdcard.busdev.busdev_u.spi.csnPin);
 }
 
 static void sdcard_deselect(void)
 {
     // As per the SD-card spec, give the card 8 dummy clocks so it can finish its operation
-    //spiTransferByte(sdcard.instance, 0xFF);
+    //spiBusTransferByte(&sdcard.busdev, 0xFF);
```

Comment:
Why is this commented out?

Reply:
I don't know; it was commented in the original code. I would prefer to reserve it until the SPI mode driver is completely overhauled.

Reply:
:-(